tttt.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python3

```

Comment:
Did you add this file accidentally to `git`? It could be useful, but in the `test` directory

Reply:
i uploaded the two last commits by accident, note the "TEMP" in the msg